### ISSUE TITLE
Duration of 0 and less than a second

### ISSUE THREAD

**Issue by yanickrochon** [moment/moment#4306]
**Description of the Issue and Steps to Reproduce:**

The value of `moment.duration(0).humanize()` yields `"a minute"` which is an odd behavior. 

**Expected or desired behavior**
```
moment.duration(0).humanize();  // "now"
moment.duration(0).humanize(true);  // "just now"
```

Rounding milliseconds would also behave the same for a duration of less than 500 ms.

**Comment by icambron** [comment]
I can reproduce that:

```js
moment.duration(0).humanize() //=> "a few seconds"
moment.duration(0).humanize(true) //=> "a few seconds ago"
```

Do you have your humanization rules customized in some way?

**Comment by yanickrochon** [comment]
This is odd, I just tried it again, and I can confirm that it yields the results you commented.

Nonetheless, `"a minute"` or `"a second"` does not affect too much of the matter I wish to address with this issue; a moment of `0` is neither a few seconds as it is a few moment; it's **now**. 

There is such inconsistency with the value of `moment.duration(0)` that makes things odd, such that

```
moment.duration(0, 'minutes').humanize());  // "a few seconds"
moment.duration(1, 'minutes').humanize());  // "a minute"
moment.duration(2, 'minutes').humanize());  // "2 minutes"
moment.duration(0, 'seconds').humanize();  // "a few seconds"
moment.duration(1, 'seconds').humanize();  // "a few seconds"
moment.duration(2, 'seconds').humanize();  // "a few seconds"
```

The value of `0` should be `"now"` and `" a few seconds"` should not be `"1 second"` for 1 second, since it's a single second!

**Comment by icambron** [comment]
It's just that the first threshold is for "a few seconds", without separate cases for "a second" "no seconds". I think the original design had in mind the case where a web page said something like "Posted a few seconds ago", where the difference between "now" and "a few seconds" didn't mean anything. I certainly get that there are applications in which you definitely want that level of precision in your strings, but I'm also pretty confident we're not going to add that feature (would require us to get "now" and "a second" in a zillion languages).

TLDR: this is a reasonable feature request we're unlikely to fulfill

**Comment by yanickrochon** [comment]
So, this is a "we don't care about this" answer, is that it?

Considering translations, you could simply add the relativeTime data set as `a few seconds` by default and allow people to actually define it, then

I don't buy that excuse.
